### ISSUE TITLE
Can't add liquidity for invalid current price

### ISSUE THREAD

**Issue by YumYumToken** [Uniswap/interface#6433]
**Bug Description**
If someone added liquidity with one token for out of range price, then, others can't add liquidity later.
In this case later liquidity providers can only add one token and not two tokens to pool.

**Steps to Reproduce**

1. Go to pools
2. Click on "+ New Position"
3. Select out range price 
4. You can see following message for one token "The market price is outside your specified price range. Single-asset deposit only."
5. Add liquidity with only one token
6. Later users can't provide two token liquidity any more.

**Expected Behavior**
Adding liquidity with only one token should be disabled and users can always add liquidity with both tokens  even when outside in current price

**Comment by halgurdh** [comment]
Yeah im having about the same issue on Arbitrum chain, cant add liquidity because of the range is not loading
